tests/tests.go

```diff
@@ -109,35 +109,31 @@ type controller struct {
 }
 
 func getTestPort(testName string) int {
-	// The port should be incremented by 3 for bytebase, Postgres, and GitLab, unless documented otherwise.
-	switch testName {
-	case "TestServiceRestart":
-		return 1234
-	case "TestSchemaAndDataUpdate":
-		return 1237
-	case "TestVCS":
-		return 1240
-	case "TestTenant":
-		return 1243
-	case "TestTenantVCS":
-		return 1246
-	case "TestTenantDatabaseNameTemplate":
-		return 1249
-	// TestGhostSchemaUpdate uses 4 ports in total. The additional one is for a MySQL instance.
-	case "TestGhostSchemaUpdate":
-		return 1252
-	case "TestBackupRestoreBasic":
-		return 1256
-	case "TestPITR":
-		return 1259
-	case "TestTenantVCSDatabaseNameTemplate":
-		return 1262
-	case "TestBootWithExternalPg":
-		return 1265
-	case "TestSheetVCS":
-		return 1269
-	case "NEXT":
-		return 1272
+	configs := []struct {
+		name  string
+		ports int
+	}{
+		// If the test starts a bytebase server, then we should allocate 3 ports for bytebase, Postgres, and GitLab, unless documented otherwise.
+		{name: "TestServiceRestart", ports: 3},
```

Comment:
maybe we can just allocate 4 ports for all of them. If a test uses more than 4, upgrade the number to 5 later?

Reply:
Do you mean this?
https://github.com/bytebase/bytebase/pull/1384/commits/c1ac1af78802e22075ed84a9cb324b71d33c776f